content/post/house-prices-in-king-county.Rmd

```diff
@@ -16,6 +16,17 @@ lgr::get_logger("mlr3")$set_threshold("warn")
 lgr::get_logger("mlr3tuning")$set_threshold("warn")
 ```
 
+The use-case illustrated below touches on the following concepts:
+
+- Data preprocessing
+- [Task](https://mlr3book.mlr-org.com/tasks.html)
+- [Fitting a learner](https://mlr3book.mlr-org.com/train-predict.html)
+- [Resampling](https://mlr3book.mlr-org.com/resampling.html)
+- [Tuning](https://mlr3book.mlr-org.com/tuning.html)
```

Comment:
It's a bit weird to link these "concepts". Which gallery example would not touch all of these parts?

Reply:
The question here is: Do we want an overview of things we assume the reader is familiar with?

I can see a few gallery examples that basically only deal with `paradox` and `mlr3`, which do not require resampling/tuning at all.

Reply:
I mean this gallery is just an informal collection of examples. Each author can do as he likes.

Reply:
Jup. I have no hard feelings here, just felt it might be useful. Can also gladly remove it.